### ISSUE TITLE
Ubuntu 18.04 does not have standard support any more, should we remove it?

### ISSUE THREAD

**Issue by gaborcsardi** [rstudio/r-builds#167]
According to https://wiki.ubuntu.com/Releases

**Comment by glin** [comment]
Yes, we should remove it. We usually do this as part of EOLing distros for Package Manager binary support, but haven't gotten to Ubuntu 18 yet. And also, I think removing Ubuntu 18 is blocked by https://github.com/rstudio/r-builds/issues/165 if that's still an issue.

**Comment by gaborcsardi** [comment]
I merged the Debian 12 PR, so we'll probably see it there?
